pkgs/by-name/co/construct/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, boost
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "construct";
+  version = "0-unstable-2024-01-16";
```

Comment:
```suggestion
  version = "unstable-2024-01-16";
```

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
I see, that's new convention: https://github.com/NixOS/nixpkgs/commit/cb89165fb9a5e80a05356af4d75ed826d7d417e0.

Reply:
the only issue is that unstableGitupdater can't deal with those yet, so for derivations that use unstableGitupdater I leave old format

see
https://github.com/NixOS/nixpkgs/pull/280501